frame/vesting/src/lib.rs

```diff
@@ -249,6 +251,10 @@ impl<T: Trait> VestingSchedule<T::AccountId> for Module<T> where
 	/// If there already exists a vesting schedule for the given account, an `Err` is returned
 	/// and nothing is updated.
 	///
+	/// On success, a linearly reducing amount of funds will be locked. In order to realise any
```

Comment:
These docs should be added to the trait definition :) 

And I think it would be good to document that this doesn't creates any currency.

Reply:
i'll put it in if you like, but nothing about the function name or existing docs state anything about adding "currency" or "balance". the name is "add_vesting_schedule", implying a vesting schedule is added, not any funds.

Reply:
Yeah that is correct, I'm probably just confused of the old implementation.

However, it would be nice to put your doc update to the trait definition.